src/OAuth/Tokens/SessionTokenStorage.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Penneo\SDK\OAuth\Tokens;
+
+class SessionTokenStorage implements TokenStorage
```

Comment:
Be aware that the main user of this SDK is forms and I'm not sure we are using sessions there (Again, I don't know if this will be used in forms at all)

Reply:
Could you elaborate on `forms`? I'm not aware of that use-case and would like to learn more. It's possible that OAuth doesn't work for that use-case, but either way, users aren't forced to store things in the session, they can implement whatever storage mechanism they want by using the `TokenStorage` interface. I've added this implementation `SessionTokenStorage` just because it implements a common usage pattern for storing transient user data in PHP.